contrib/inventory/foreman.ini

```diff
@@ -116,6 +127,8 @@ group_patterns = ["{app}-{tier}-{color}",
 group_prefix = foreman_
 # Whether to fetch facts from Foreman and store them on the host
 want_facts = True
+# Whether to create Ansible groups for host collections
+want_hostcollections = False
```

Comment:
Getting Host Collections isn't a expensive API call (unlike Facts), if you want to remove the option and make it a default that's all good. The only issue is that possibly the inventory would not work with just Foreman (without Katello) though.. mind to leave a comment about that?